kafkakewl-metrics/src/main/scala/com/mwam/kafkakewl/metrics/domain/KafkaConsumerGroupMetrics.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Marshall Wace <opensource@mwam.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.mwam.kafkakewl.metrics.domain
+
+enum ConsumerGroupStatus(val severity: Int):
```

Comment:
shouldn't scalafmt change this to the braces-style?

Reply:
It hasn't, and this is how I saw the docs from online. I didn't know it could be done using braces-style, I've fixed this up now